### PR TITLE
[New Rule] Microsoft 365 - Mass download by a single user

### LINKED ISSUE #1318: [New Rule] Microsoft 365 - Mass download by a single user
## Description
Identifies when Microsoft Cloud App Security reported when a single user performs more than 50 downloads within 1 minute.

### Target indexes

`filebeat-*, logs-o365*`

### Platforms

Microsoft 365

### Query

```
event.dataset:o365.audit and event.provider:SecurityComplianceCenter and event.category:web and 
event.action:"Mass download by a single user" and event.outcome:success
```

### **False Positives**
- 
- 

### MITRE

| Tactic | Technique ID | Technique Name | Sub-Technique Name |
| ------ | ------------ | -------------- | ------------------ |
|        |              |                |                    |
|        |              |                |                    |

### References
- https://docs.microsoft.com/en-us/cloud-app-security/anomaly-detection-policy
- https://docs.microsoft.com/en-us/cloud-app-security/policy-template-reference

### PR DESCRIPTION
**Pull request by austinsonger**
## Issues

Resolves #1318

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by austinsonger** [comment]
It looks like I created these rules before elastic is able to trigger the detection. Should I just leave this open until the integration happens?

https://github.com/elastic/integrations/issues/1131

File: rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/07/15"
+maturity = "development"
+updated_date = "2021/07/15"
+
```
Comment (w0rk3r) [inline rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml:5]: Missing Integration

File: rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+author = ["Austin Songer"]
+description = """
+Identifies when Microsoft Cloud App Security reported when a single user performs more than 50 downloads within 1 minute.
+"""
+false_positives = ["Unknown"]
+from = "now-30m"
+index = ["filebeat-*", "logs-o365*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Microsoft 365 - Mass download by a single user"
```
Comment (w0rk3r) [inline rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml:16]:
```suggestion
name = "Microsoft 365 Mass download by a single user"
```

File: rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+query = '''
+event.dataset:o365.audit and event.provider:SecurityComplianceCenter and event.category:web and event.action:"Mass download by a single user" and event.outcome:success
+'''
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+name = "Data Destruction"
+reference = "https://attack.mitre.org/techniques/T1485/"
+id = "T1485"
```
Comment (w0rk3r) [inline rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml:41]: Can't see a case for data destruction on this one

File: rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+[rule.threat.tactic]
+name = "Impact"
+reference = "https://attack.mitre.org/tactics/TA0040"
+id = "TA0040"
```
Comment (w0rk3r) [inline rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml:46]: I think we can remove the technique here and change the tactic to Exfiltration: https://attack.mitre.org/tactics/TA0010/

**Review by w0rk3r — CHANGES_REQUESTED** [review]
Leaving some comments/requests here. Thanks for the contribution!

**Comment by w0rk3r** [comment]
Hey @austinsonger, any updates on this one?

**Pull request merged**